WatchFaceFormat/validator-plugin/plugins/src/main/kotlin/WffValidatorPlugin.kt

```diff
@@ -29,30 +29,42 @@ import org.gradle.kotlin.dsl.register
 const val ASSEMBLE_DEBUG_TASK = "assembleDebug"
 const val BUNDLE_DEBUG_TASK = "bundleDebug"
 const val VALIDATE_TASK = "validateWff"
+const val MEMORY_FOOTPRINT_TASK = "memoryFootprint"
 const val DOWNLOAD_VALIDATOR_TASK = "downloadWffValidator"
+const val DOWNLOAD_MEMORY_FOOTPRINT_TASK = "downloadMemoryFootprint"
 const val INSTALL_TASK = "validateWffAndInstall"
 
-// TODO move from here
 private const val VALIDATOR_URL =
     "https://github.com/google/watchface/releases/download/release/dwf-format-2-validator-1.0.jar"
 private const val VALIDATOR_PATH = "validator/validator.jar"
+private const val MEMORY_FOOTPRINT_URL =
+    "https://github.com/google/watchface/releases/download/latest/memory-footprint.jar"
```

Comment:
should "latest" be extracted and made just a default, and allow using a tagged version.

Non reproducible builds seems worrying, and also latest may change but won't get cached correctly.

Reply:
Apologies, have changed the default tag to "release" and added a configuration option to change it (if needed).